examples/regression/script.jl

```diff
@@ -0,0 +1,196 @@
+# A recreation of <https://gpflow.readthedocs.io/en/master/notebooks/advanced/gps_for_big_data.html>
+
+# # Stochastic Variational Regression
+#
+# In this example, we show how to construct and train the stochastic variational
+# Gaussian process (SVGP) model for efficient inference in large scale datasets.
+# For a basic introduction to the functionality of this library, please refer to
+# the [User Guide](@ref).
```

Comment:
link doesn't seem to be working (same for classification example)

Reply:
Yeah - I can't figure out why...

Reply:
#36